### PR TITLE
Update notes import

### PR DESCRIPTION
**Pull request by pompan129**
- updated import documentation. 
  - corrected accepted import file types
  - corrected location of importspecifications links
- new images for notes section (corrected spelling error)

### CODE REVIEW

File: _docs/tools/data-tools.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ Import data only for your current agency. You can change agencies using the **Cu
 ![Current Agency Box]({{ site.baseurl }}/assets/images/docs/current-agency.PNG)
 
 - **Import Type**: What type of data are you importing? Only one selection is possible.
-- **File**: Choose the file to upload from your desktop. Note that the file must be one of Microsoft Excel (x.xls), OpenDocument Spreadsheet (x.ods), Comma Separated Values (x.csv), or HTML (x.html)
-  - Note that the file must follow the import rules outlined to the box on the side for the specific _Import Type_ being used in order for the import to be successful.
+- **File**: Choose the file to upload from your desktop. Note that the file must be one of Microsoft Excel (.xls or .xlsx) or Comma Separated Values (.csv).
+  - Note that the file must follow the import rules for the specific _Import Type_ being used in order for the import to be successful. Import specifications can be found by following the links in the information panel below.
+
+  ![Bulletin Board Notes]({{ site.baseurl }}/assets/images/docs/info-panel.png)
```
Comment (jodysimpson) [inline _docs/tools/data-tools.md:17]: Please correct the alt tag here.  This image is the file upload information panel, not `Bulletin Board Notes`

File: _docs/tools/data-tools.md [hunk 2]
```diff
@@ -11,8 +11,10 @@ Import data only for your current agency. You can change agencies using the **Cu
 ![Current Agency Box]({{ site.baseurl }}/assets/images/docs/current-agency.PNG)
 
 - **Import Type**: What type of data are you importing? Only one selection is possible.
-- **File**: Choose the file to upload from your desktop. Note that the file must be one of Microsoft Excel (x.xls), OpenDocument Spreadsheet (x.ods), Comma Separated Values (x.csv), or HTML (x.html)
-  - Note that the file must follow the import rules outlined to the box on the side for the specific _Import Type_ being used in order for the import to be successful.
+- **File**: Choose the file to upload from your desktop. Note that the file must be one of Microsoft Excel (.xls or .xlsx) or Comma Separated Values (.csv).
+  - Note that the file must follow the import rules for the specific _Import Type_ being used in order for the import to be successful. Import specifications can be found by following the links in the information panel below.
```
Comment (jodysimpson) [inline _docs/tools/data-tools.md:15]: Please include the links to the import specifications instead of telling them to follow the links in the image.

**Review by jodysimpson — CHANGES_REQUESTED** [review]

File: _docs/tools/data-tools.md (hunk 1 above)
Comment (pompan129) [reply _docs/tools/data-tools.md:17]: ooh. thanks

File: _docs/tools/data-tools.md (hunk 2 above)
Comment (pompan129) [reply _docs/tools/data-tools.md:15]: cool

**Pull request merged**